lib/manager/gradle/shallow/extract.ts

```diff
@@ -64,6 +76,41 @@ export async function extractAllPackageFiles(
         const { vars, deps } = parseProps(content, packageFile);
         updateVars(vars);
         extractedDeps.push(...deps);
+      } else if (isTOMLFile(packageFile)) {
+        // Implement TOML file parsing and extraction
+        const tomlContent = parse(content);
```

Comment:
```suggestion
        const tomlContent = parse(content) as GradleCatalog;
```

Please define `GradleCatalog` interface at `./types.ts`.

Reply:
Done in 09c59be